### PR TITLE
Bug 1741176: Enhance host maintenance status to be tracked using maintenance status

### PR DESCRIPTION
**Pull request by jtomasek**
Enhance host maintenance status to be tracked using maintenance status
Track maintenance progress in maintenance popover showing percentage
based on maintenance status information
Improve the stop maintenance modal to not include maintenance name

https://bugzilla.redhat.com/show_bug.cgi?id=1741176
https://bugzilla.redhat.com/show_bug.cgi?id=1737901

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx [hunk 1]
```diff
@@ -39,33 +32,17 @@ export const SetNodeMaintanance = (
   callback: () => startNodeMaintenanceModal({ nodeName }),
 });
 
-export const RemoveNodeMaintanance = (
+export const RemoveNodeMaintenance = (
```
Comment (knowncitizen) [inline frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx]: Took me a second of staring to realize the change, ha!

**Comment by ahrechanychenko** [comment]
@jtomasek 
Hi Jiri,
I tested pull locally and got next issues:
1) Status pop up didn't show when click for Stopping Maintenance status. But it works for Starting Maintenance|Under Maintenance status.
2) When click to Starting Maintenance status, to Show Workload drop-down list - it doesn't allow to scroll or close drop-down list
Screenshot from 2019-08-23 13-51-48
Screenshot from 2019-08-23 14-01-06
Screenshot from 2019-08-23 14-02-30

**Comment by jtomasek** [comment]
> 1. Status pop up didn't show when click for Stopping Maintenance status. But it works for Starting Maintenance|Under Maintenance status.
I believe it is not necessary to provide a popover for stopping the maintenance as that operation takes very short time (nodeMaintenance CR just gets deleted)

> 2. When click to Starting Maintenance status, to Show Workload drop-down list - it doesn't allow to scroll or close drop-down list
I've moved the pods list into a fixed height scrollable virtualized list, so the popover height does not expand with large number of pending pods.

**Pull request merged**